src/main/java/com/capitalone/dashboard/collector/DefaultArtifactoryClient.java

```diff
@@ -1,14 +1,9 @@
 package com.capitalone.dashboard.collector;
 
 import com.capitalone.dashboard.client.RestClient;
-import com.capitalone.dashboard.model.ArtifactItem;
-import com.capitalone.dashboard.model.ArtifactoryRepo;
-import com.capitalone.dashboard.model.BaseArtifact;
-import com.capitalone.dashboard.model.BinaryArtifact;
-import com.capitalone.dashboard.model.ServerSetting;
+import com.capitalone.dashboard.model.*;

```

Comment:
can you please remove this kind of import by fixing your ide settings. we want specific imports